crowdin.yml

```diff
@@ -4,9 +4,7 @@ files:
     translation: /pages/%two_letters_code%/**/%original_file_name%
     dest: /**/%original_file_name%
     content_segmentation: 0
-    skip_untranslated_files: true
```

Comment:
Turns out these options are not supported by Crowdin's GitHub integration, so removing. I enabled them at the project level instead (we can't get file type granularity that way, but that's OK).